src/test/java/com/google/javascript/gents/multiTests/module_imports/export_sideeffect.ts

```diff
@@ -1,2 +1,5 @@
-export {};
+export {
+  //  Empty export added to coerce the file to be a module, since it does not
```

Comment:
uber-nit: maybe make it a line comment, and make it a bit terser?

`export {/* gents: force this file to be an ES6 module (no imports or exports) */};'`